go-example/metrics/gooteltest/README.md

```diff
@@ -18,7 +22,7 @@ See the sample config file in example.yaml.
 | FieldName | Description |
 | --------- | ----------- |
 | collectPeriod | This is the amount of time to wait before sending the next group of metrics. Default is 10s |
-| otelCollector | This is the host and port that the OTEL collector is listening on for metrics. Default is localhost:4317 |
+| aggregationTemporalitySelector | TemporalitySelector used for selecting aggregation(i.e., Cumulative vs. Delta aggregation). If not specified otherwise, exporter will use a cumulative temporality selector. |
```

Comment:
It might be good to throw an error if the user misspells 'cumulative' or 'delta' in the config file. But if they don't specify it then default to 'cumulative'.  To do this, change the fixDefaults() method on Config by adding

```
if c.AggregationTemporalitySelector == "" {
    c.AggregationTemporalitySelector = aggregationCumulative
}
```

Note that you want to define aggregationCumulative and aggregationDelta as constants.  Then in the checkConfig() function, throw an error if AggregationTemporalitySelector is not aggregationCumulative or aggregationDelta.